samples/builtins/README.md

```diff
@@ -0,0 +1,18 @@
+# Builtin Modules
+
+This directory contains comprehensive sample of using builtin javascript modules.
+Builtin modules extend workerd functionality for all workers hosted by a given server.
+
+This example demonstrates following features:
+- using builtins to provide new user-level api
+- using internal builtin modules to hide implementation details from the user
+
+The sample will be extended as more functionality is implemented.
+
+## Demo bundle
+
+The bundle is defined in [alibaba-bundle.capnp](alibaba-bundle.capnp) and specified modules
```

Comment:
FWIW when I see "alibaba" I think of the tech company Alibaba rather than the Arabian Nights character Ali Baba, which threw me off for a bit. I was wondering if this sample was using some Alibaba API or something. Might be worth adding a few words explaining the reference? Not everyone is familiar with the story.

Reply:
Reworked the example to be a fictional burrito shop api. Let's see if it less confusing and serves me well to demonstrate wrapped bindings.